release-notes/Cube/Cube_Feature_Release_10.5/Cube_10.5.5.md

```diff
@@ -12,49 +12,130 @@ uid: Cube_Feature_Release_10.5.5
+
+![View impact icon in the Surveyor](~/develop/images/EPM_Surveyor_alarm_icon.png)
+
+##### Support for multiple front-end manager elements
+
+Previously, the *CPEIntegration* soft-launch feature only allowed one front-end manager element. Now, multiple front-end elements are supported in one DataMiner System.
+
+To configure this, **make sure the system type for each front-end manager is unique**, as it is the system type that now will be used to trace EPM objects back to their respective front-end managers. If you for example have an "HFC" and an "IOT" front-end EPM Manager in your system, these cannot both have a "Location" cell. Instead, you can prefix this, e.g. "HFC_Location" and "IOT_Location", respectively.
+
+In addition, the front-end and back-end elements for the **same technology** must use the **same DataMiner protocol and version**, as otherwise this may result in incorrect linking in the DataMiner Cube UI.
+
+If these conditions are met, to start using the front-end EPM Managers, first define them in System Center via *System settings* > *EPM config* or via the cogwheel icon in the Topology app.
 
 > [!IMPORTANT]
->
-> - Each of the frontend manager connectors must have a different *SystemType*.
-> - All frontend manager elements must use the same version of the same connector.
-> - All backend manager elements must use the same version of the same connector.
-> - All EPM objects must be linked using both *SystemName* and *SystemType*. For example, in Visual Overview, you can link a shape to an EPM object by means of the *SystemName* and *SystemType* data fields.
+> Because of this change, some **configuration changes may be required in systems that were previously using the *CPEIntegration* soft-launch option**. Some existing solutions may not have a unique system type per EPM Manager protocol, or they may have front-end and back-end manager elements with different protocols or protocol versions, which will no longer work correctly.
+
+##### Customization of EPM integration data pages
+
+By default, the table and column parameters linked to an EPM object will automatically be positioned on data pages. The name of a page will match the description of the table, and the parameters will be displayed in the order defined in the column definitions of the table, in two columns.
+
+You can change the name of a page by giving the table a position on a page that has a name prefixed with `CPEIntegration_`, e.g. *CPEIntegration_MyCustomPage*. To hide a complete table, add a table position on the special page *CPEIntegration_Hidden*. Note that pages with the `CPEIntegration_` prefix will not be shown on regular element cards.
```

Comment:
@MichielVT-SLC This is based on the old wiki, but I couldn't find a corresponding RN. Could you confirm if this is indeed OK?

Reply:
Is this what you mean?
```suggestion
You can change the name of a page by giving a table column parameter a position on a page that has a name prefixed with `CPEIntegration_`, e.g. *CPEIntegration_MyCustomPage*. To hide a table column completely, add the table column position on the special page *CPEIntegration_Hidden*. Note that pages with the `CPEIntegration_` prefix will not be shown on regular element cards.
```